src/style.css

```diff
@@ -1 +1,50 @@
-/* Styles go here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap");
+
+header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav-list {
+  display: flex;
+  padding: 0;
+  margin-right: 40px;
+}
+
+.nav__link {
+  display: block;
+  padding: 20px 0;
+  font-family: "Roboto", sans-serif;
+  text-decoration: none;
+  color: black;
+  font-weight: 500;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.img-logo {
+  margin: 10px 0 0 20px;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  display: block;
+  height: 5px;
+  width: 100%;
+  bottom: 0;
+  left: 20px;
```

Comment:
you added is-active class for nav__list and nav__link. You need to add active class just for the link. And you don’t need add left property